.github/workflows/ci.yml

```diff
@@ -148,9 +132,13 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@nightly
       - run: ./ci/run_ci.sh rust
+
   cpp:
     name: C++ CI
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Does this include bont Mac Intel and Mac silicon?

Reply:
No. Currently it's x86.

Reply:
Actually it's a hard problem for arm64-based GitHub Action runners (it is currently not supported by GitHub official runners, but supported only in self-hosted runners). cc @tisonkun may have some information.

Reply:
We never release fury python for Mac M1/M2, seems it's not easy to automate it. It's beyoud the scope of this PR, we can discuss it in #812